sdk/storage/Azure.Storage.sln

```diff
@@ -0,0 +1,131 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28803.352
+MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
also differs from above

Reply:
I think these are fine to leave in because they're standard VS sln properties.